src/immutably-array/move.test.js

```diff
@@ -0,0 +1,70 @@
+import test from 'tape';
+
+import {move} from './move';
+
+test('immutably-array / move / move item in an empty array', (testCase) => {
+    testCase.doesNotThrow(() => {
+        const input = [];
+        const resultOutput = move(input, null, 0, 1);
+        const expectedOutput = [];
+
+        testCase.deepEqual(resultOutput, expectedOutput);
+        testCase.notEqual(input, resultOutput);
```

Comment:
I'd expect `equal` in this case. I believe the line I suggested to add would take care of this.